### PR TITLE
New package: NetworkManager-strongswan-1.4.4

### PR DESCRIPTION
**Pull request by DBLouis**

### CODE REVIEW

File: srcpkgs/strongswan/template [hunk 1]
```diff
@@ -7,9 +7,9 @@ build_style=gnu-configure
 configure_args="--disable-static --enable-blowfish --enable-curl
  --enable-eap-radius --enable-eap-mschapv2 --enable-eap-md5 --enable-led
  --enable-ha --enable-dhcp --enable-mediation --enable-soup --disable-des
- --enable-chapoly"
+ --enable-chapoly --enable-nm"
```
Comment (Hoshpak) [inline srcpkgs/strongswan/template]: You should bump the revision of the template so the package gets rebuilt with your changes.

Comment (DBLouis) [reply srcpkgs/strongswan/template]: Of course

**Comment by DBLouis** [comment]
Why is the build failing?

**Comment by jnbr** [comment]
> Why is the build failing?

There were some problems with libressl. Rebasing your commit on master should fix the travis builds.

**Pull request merged**